src/types.ts

```diff
@@ -112,6 +123,37 @@ export interface PaletteColor {
   color: string | null;
 }
 
+interface TextAlign {
+  auto: boolean;
+  align: string;
+}
+
+export interface ExtendedNxAttrExprInfo extends EngineAPI.INxAttrExprInfo {
+  id: string;
+  qMin: number;
+  qMax: number;
+  qContinuousAxes: boolean;
+  qIsCyclic: boolean;
+  qFallbackTitle: string;
+}
+
+export interface ExtendedNxDimensionInfo extends Omit<EngineAPI.INxDimensionInfo, 'qAttrExprInfo'> {
+  textAlign: TextAlign;
+  qAttrExprInfo: ExtendedNxAttrExprInfo[];
+}
+
+export interface ExtendedNxMeasureInfo extends EngineAPI.INxMeasureInfo {
+  textAlign: TextAlign;
+  qAttrExprInfo: ExtendedNxAttrExprInfo[];
+}
+
+export interface HyperCube extends Omit<EngineAPI.IHyperCube, 'qDimensionInfo' | 'qMeasureInfo'> {
+  qColumnOrder: number[];
```

Comment:
does this not exist on the EngineAPI.IHyperCube ?

Reply:
No, qColumnOrder does not exist on the EngineAPI.IHyperCube

Reply:
hmm surprising